README.md

```diff
@@ -68,6 +68,11 @@ An absolute path in which nginx will generate and serve the CBOR-encoded certifi
 But make sure that the OCSP responder for the certificate is accessible from your nginx server to get OCSP responses.
 This directive is optional.
 
+#### sxg\_expires\_seconds
```

Comment:
Perhaps "expiry" is slightly better?  (not sure)

Reply:
done.